includes/admin/functions.php

```diff
@@ -194,3 +194,28 @@ function amp_bootstrap_admin() {
 	$site_health = new SiteHealth();
 	$site_health->init();
 }
+
+/**
+ * Whether to activate the new onboarding feature.
+ *
+ * @return bool
+ */
+function amp_should_use_new_onboarding() {
+	global $wp_version;
+
+	if ( version_compare( $wp_version, '5.0', '<' ) ) {
```

Comment:
Minor thing, but instead of getting the global variable you could rather do:

```suggestion
	if ( version_compare( get_bloginfo( 'version' ), '5.0', '<' ) ) {
```